MAUI/ComboBox/Getting-Started.md

```diff
@@ -246,3 +246,9 @@ The following gif image illustrates the result of the above code:
 ![.NET MAUI ComboBox choose item using editing](Images/GettingStarted/EditableMode.png)
 
 You can find the complete getting started sample of .NET MAUI ComboBox from this [link.](https://github.com/SyncfusionExamples/maui-combobox-samples)
+
+## Text
+
+The [Text]() property used to gets the user-submitted text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode. The default value of the `Text` property is `string.Empty`.
```

Comment:
The [Text]() property used to get the user-submitted text in the [SfComboBox](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfComboBox.html) editable mode. The default value of the `Text` property is the `string.Empty`.